### PR TITLE
Prepare to write TRAN to init file.

### PR DESCRIPTION
**Pull request by joakim-hove**
Minor refactoring & cleanup of the writing of INIT files.

### CODE REVIEW

File: tests/test_EclipseWriter.cpp [hunk 1]
```diff
@@ -262,27 +264,35 @@ BOOST_AUTO_TEST_CASE(EclipseWriterIntegration)
...
-        eclWriter.writeInit();
+        std::vector<double> tranx(3*3*3);
+        std::vector<double> trany(3*3*3);
+        std::vector<double> tranz(3*3*3);
+        std::vector<data::CellData> simProps{{"TRANX" , UnitSystem::measure::transmissibility, tranx},
+                                             {"TRANY" , UnitSystem::measure::transmissibility, trany},
+                                             {"TRANZ" , UnitSystem::measure::transmissibility, tranz}};
+
+        ta.setStore( true );
+        std::cout << "Working in: " << ta.getCwd() << std::endl;
```
Comment (pgdr) [inline tests/test_EclipseWriter.cpp:280]: debug or production?

File: opm/output/eclipse/EclipseWriter.hpp [hunk 2]
```diff
@@ -51,12 +51,13 @@ class EclipseWriter {
...
                   const NNC& );
 
+
     /**
      * Write the static eclipse data (grid, PVT curves, etc) to disk.
      *
      * If NNC is given to the constructor, writes TRANNNC keyword.
      */
-    void writeInit();
+    void writeInit(const std::vector<data::CellData>& simProps = {}, const NNC& nnc = NNC());
```
Comment (pgdr) [inline opm/output/eclipse/EclipseWriter.hpp:60]: Could you add a comment also on the use of `simProps`?

**Comment by pgdr** [comment]
I am ready to merge this when the minor comments are addressed.

**Comment by joakim-hove** [comment]
Hmmm - I ended up doing the fixup here instead: https://github.com/OPM/opm-output/pull/51

**Comment by jokva** [comment]
`writeINITFile` and `writeEGRIDFile` doesn't modify `EclipseWriter` at all, so I'd personally prefer for them to be static'd in the source file, rather than being exposed to the header. Makes the header slightly simpler, exposes fewer details (especially since we already use private implementation here) and makes its actual input-dependencies obvious via arguments.

Other than that it looks good.

**Comment by joakim-hove** [comment]
I'm closing this; discussion should continue here: #51

**Pull request closed without merging**